### PR TITLE
Features/omojola faith/email with map prompt

### PR DESCRIPTION
**Pull request by codemutatio**
**What does this PR do**

1. Send a mail to the emergency response team when an accident report is submitted.

2. Integrated google static map api to generate a map view of the accident location.

### CODE REVIEW

File: server/controllers/reportControllers.js [hunk 1]
```diff
@@ -14,6 +23,17 @@ const Reports = {
...
+            sendMail('lifeaidd@gmail.com',
+                'Emergency report',
+                'faithomojola@yahoo.com',
+                `<b>Emergency </b> <br></br><br></br>
+                <img src="https://maps.googleapis.com/maps/api/staticmap?
+                center=${queryText.coords[0]},${queryText.coords[1]}
+                &zoom=20&size=500x500
+                &markers=color:red%7Clabel:S%7C${queryText.coords[0]},${queryText.coords[1]}
+                &key=${MAP_API_KEY}"> <br></br><br></br> 
+                ${queryText.audioUrl}`);
```
Comment (enyeInc) [inline server/controllers/reportControllers.js]: put this logic in a function; `line 29 - 35`

File: test/aid.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable sort-keys */
```
Comment (enyeInc) [inline test/aid.js]: why is a test for aids in a PR for email logic? separation of concerns

File: server/controllers/reportControllers.js [hunk 3]
```diff
@@ -1,28 +1,47 @@
...
         const { public_id: audioId, url: audioUrl } = await req.file;
         const queryText = { ...req.body, audioId, audioUrl };
+        const { coords } = queryText;
 
         try {
             const report = await db.create(queryText);
+            sendMail('lifeaidd@gmail.com',
+                'Emergency report',
+                'faithomojola@yahoo.com',
+                messageBody(coords[0], coords[1], audioUrl));
```
Comment (Way2nnadi) [inline server/controllers/reportControllers.js]: this should be `messageBody(coords, audioUrl)`. Let the function determine how to use the coords that you pass it

Comment (codemutatio) [reply server/controllers/reportControllers.js]: ok

**Pull request closed without merging**